resources/validations/ui/src/views/components/Banner.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { useActions } from '../hooks';
+
+export const Banner = () => {
+  const { getAssetUrl } = useActions();
+  return (
+    <section className="usa-banner" aria-label="Official government website">
+      <div className="usa-accordion">
+        <header className="usa-banner__header">
+          <div className="usa-banner__inner">
+            <div className="grid-col-auto">
+              <img
+                className="usa-banner__header-flag"
+                src={getAssetUrl('uswds/img/us_flag_small.png')}
+                alt="U.S. flag"
+              />
+            </div>
+            <div className="grid-col-fill tablet:grid-col-auto">
+              <p className="usa-banner__header-text">
+                An official website of the United States government
+              </p>
+              <p className="usa-banner__header-action" aria-hidden="true">
+                Here’s how you know
+              </p>
+            </div>
```

Comment:
B: Can we perhaps add also a beta or alpha banner [like that of the 10x/18F SORN DASH app](https://github.com/18F/all_sorns/blob/5bbe519183ba804e3ddfef8b29d7e6e460ef2bc5/app/views/layouts/banner.html.erb#L2-L6) to set expectations?

```html
<section class="beta-banner">
    <div class="grid-container">
    <p class="grid-col-auto"><strong>BETA SITE:</strong> We are testing a new centralized System of Records Notice dashboard.</p>
    </div>
</section>
```